### DISCUSSION TITLE
Is there anyway to make 120 people with avatar in 1 room?

### DISCUSSION THREAD

**Discussion by Takosant** [mozilla/hubs#5267] [General Questions & Discussion]
Hello , I have to do the event in Mozilla with 120 people in room. 16 rooms at the same time. right now we use 4 servers with 5.4xlarges but still have problem about sound that unstable for user. so is there anyway we can solve this problem? will upgarde server help? or can I split each room to 1 server would be better? Thank you for you answer in advance.

**Comment by pattersonbl2** [comment]
Currently, we can't run this setup.  You can have up to 50 people in a room, but this could cause performance issues.

**Comment by Dayk0** [comment]
Impossible. 50 people maximum if the room is very light and if you have very simple avatars.
Otherwise, it will be 25 people maximum with a normal and slightly complex room as well as slightly more developed avatars.
